docs/repo-docs/reference/watch.mdx

```diff
@@ -13,9 +13,19 @@ turbo watch [tasks]
 
 ## Using `turbo watch` with persistent tasks
 
-When your script has a built-in watcher, you likely don't need to use `turbo watch`. Instead, use your script's built-in watcher and mark the task as long-running using [`"persistent": true`](/repo/docs/reference/configuration#persistent).
+Persistent tasks are marked with [`"persistent": true`](/repo/docs/reference/configuration#persistent), meaning they won't exit. Because of this, they cannot be dependended on in your task graph.
```

Comment:
```suggestion
Persistent tasks are marked with [`"persistent": true`](/repo/docs/reference/configuration#persistent), meaning they won't exit. Because of this, they cannot be depended on in your task graph.
```